SingularityUI/app/controllers/RequestDetail.coffee

```diff
@@ -17,6 +17,7 @@ class RequestDetailController extends Controller
 
     templates:
         header:         require '../templates/requestDetail/requestHeader'
+        taskHistoryMsg: require '../templates/requestDetail/taskHistoryMsg'
```

Comment:
how come we're calling it `taskHistoryMsg` -- isn't this information about the request?